Formula/elftoolchain.rb

```diff
@@ -0,0 +1,151 @@
+class Elftoolchain < Formula
+  desc "Implementation of compilation tools for the ELF object format"
+  homepage "https://sourceforge.net/p/elftoolchain/wiki/Home/"
+  license all_of: ["BSD-2-Clause", "BSD-3-Clause"]
+  head "https://svn.code.sf.net/p/elftoolchain/code/trunk"
+
+  stable do
+    url "https://downloads.sourceforge.net/project/elftoolchain/Sources/elftoolchain-0.7.1/elftoolchain-0.7.1.tar.bz2"
+    sha256 "44f14591fcf21294387215dd7562f3fb4bec2f42f476cf32420a6bbabb2bd2b5"
+
+    # Avoid trying to run svnversion or git describe
+    patch :p0 do
+      url "https://raw.githubusercontent.com/macports/macports-ports/195adb123b397d5ae850d91105f668aff83e8161/devel/elftoolchain/files/patch-libelftc-make-toolchain-version.diff"
+      sha256 "2e05bcfd4944e4be72d8166b3b9ef11368f15e30319b945f81b4f32c2863ba7d"
+    end
```

Comment:
May not need this patch if we add `subversion` as build dependency as it only fails if it cannot run `svnversion`; however, the result may be some ` svn:Unversioned directory` appended to version number.

Reply:
Will probably just do this and then we can make `subversion` head-only on next release (`subversion` should already be used for HEAD but having it in environment will allow burning in correct revision number).